src/sardana/pool/poolmeasurementgroup.py

```diff
@@ -267,84 +368,82 @@ def _build_channel_defaults(self, channel_data, channel):
 
         return channel_data
 
-    def _build_configuration(self):
+    def _build_configuration(self, config=None):
```

Comment:
I don't like the idea to allow to pass here the config

Reply:
This is not the user_configuration and the idea is to have only one method to prepare and validate the configuration. We can design another implementation